src/measures/MultiS3.cpp

```diff
@@ -1,29 +1,53 @@
 #include "MultiS3.hpp"
 #include <cmath>
 
+unsigned NUM_GRAPHS;
 unsigned MultiS3::denom = 0;
-vector<uint> MultiS3::shadowDegrees;
+vector<uint> MultiS3::totalDegrees;
 
 MultiS3::MultiS3(Graph* G1, Graph* G2) : Measure(G1, G2, "ms3")
 {
+    extern char *getetv(char*);
+    char *s = getenv((char*)"NUM_GRAPHS");
+    if (s)
+    {
+        assert(1 == sscanf(s, "%u",&NUM_GRAPHS));
+    }
+    else
+    {
+        cout << "Warning: NUM_GRAPHS should be an environment variables; setting to 2 for now\n";

```

Comment:
Hmm. May be better here to assert(MULTI_PAIRWISE) is set, and if it is then die if NUM_GRAPHS is not set.